gerber/utils.py

```diff
@@ -123,6 +123,10 @@ def write_gerber_value(value, format=(2, 5), zero_suppression='trailing'):
     value : string
         The specified value as a Gerber/Excellon-formatted string.
     """
+    
+    if format[0] == float:
+        return "%f" %value
+    
```

Comment:
This would not format the float value according to requested format and zero suppression. Any particular problem you were trying to fix here?

Reply:
> This would not format the float value according to requested format and zero suppression. Any particular problem you were trying to fix here?

Nevermind, I read the rationale in the PRs description and understood now.